config/middlewares/validations.js

```diff
@@ -0,0 +1,26 @@
+const { check, validationResult } = require('express-validator/check');
+/**
+ * @description - Validate user signup
+ * @param {object} req - request body
+ * @param {object} res - response body
+ *
+ * @return {undefined} depends on user input
+ */
+exports.validateSignup = (req, res, next) => {
+  // Finds the validation errors in this request and wraps them in an object with handy functions
+  const errors = validationResult(req);
```

Comment:
@OlatunjiYso Why are we validating the entire `req` object? 🤔  
I'm sure we're only expecting users input at `req.body` and not the entire `req` object.

Reply:
Thank you @fortunee.
please, I seem not to have control over the validator API, 
I did it according to the [documentation](https://express-validator.github.io/docs/index.html)
what is being referenced eventually is `req.body.` ... what is set.